mmgotool/commands/i18n.go

```diff
@@ -4,13 +4,15 @@
 package commands
```

Comment:
I love all the code improvements made to this file. Make it a bit harder to review, but really appreciate the work on cleaning this up. Thanks!